Protocol.md

```diff
@@ -875,3 +878,142 @@ assume it is set and act accordingly.
+
+As a Requester or Responder __sends__ REQUEST, CANCEL, or RESPONSE frames, it maintains a __position__ of that frame within the connection in that direction. This is a 64-bit value that starts at 0. As a Requester or Responder __receives__ REQUEST, CANCEL, or RESPONSE frames, it maintains an __implied position__ of that frame within the connection in that direction. This is also a 64-bit value that starts at 0.
+
+The reason this is “implied” is that the position is not included in each frame and is inferred simply by the message being sent/received on the connection in relation to previous frames.
+
+This position will be used to identify the location for resuming operation to begin.
+
+Frame types outside REQUEST, CANCEL, ERROR, and RESPONSE do not have assigned (nor implied) positions.
+
+### Client Lifetime Management
+
+Client lifetime management for servers must be extended to incorporate the length of time a client may successfully attempt resumption. A server may reject resumption if the client has been gone too long. The means of client lifetime management are totally up to the implementation.
+
+### Resume Operation
+
+All ERROR frames sent MUST be CONNECTION_ERROR or CONNECTION_ERROR_NO_RETRY error code.
+
+Client side resumption operation starts when the client desires to try to resume and starts a new transport connection. The operation then proceeds as the following:
+
+* Client sends RESUME frame. The client must NOT send any other frame types until resumption succeeds. The RESUME Identification Token MUST be the token used in the original SETUP frame. The RESUME Last Received Position field MUST be the last successfully received implied position from the server.
+* Client waits for either a RESUME_OK or ERROR frame from the server.
+* On receiving an ERROR frame, the client MUST NOT attempt resumption again if the error code was CONNECTION_ERROR_NO_RETRY.
+* On receiving a RESUME_OK, the client:
+    * MUST assume that the next REQUEST, CANCEL, ERROR, and RESPONSE frames have an implied position commencing from the last implied positions
+    * MAY retransmit *all* REQUEST, CANCEL, ERROR, and RESPONSE frames starting at the RESUME_OK Last Received Position field value from the server.
+    * MAY send an ERROR frame indicating the end of the connection and MUST NOT attempt resumption again
+
+Server side resumption operation starts when the client sends a RESUME frame. The operation then proceeds as the following:
+
+* On receiving a RESUME frame, the server:
+    * MUST send an ERROR frame if the server does not support resuming operation. This is encoded as a 0 for the Ignore bit in the flags.
```

Comment:
I don't understand this part: "This is encoded as a 0 for the Ignore bit in the flags."

What does the Ignore bit have to do with this?